hat/assets/js/apps/Iaso/domains/orgUnits/components/OrgUnitInfosComponent.js

```diff
@@ -210,77 +223,82 @@ const Actions = (
     ];
 };
 
-const OrgUnitCreationDetails = ({ orgUnit, formatMessage, classes }) => {
+const Row = ({ label, value }) => {
+    const classes = useStyles();
+    return (
+        <TableRow>
+            <TableCell className={classes.leftCell}>{label}</TableCell>
+            <TableCell>{value}</TableCell>
+        </TableRow>
+    );
+};
+
+const OrgUnitCreationDetails = ({ orgUnit, formatMessage }) => {
```

Comment:
Not blocking but I don't see why we pass `formatMessage` in the props. 
If you feel like doing a bit of cleanup, it could be replace by `useSafeIntl` as is done elsewhere

Reply:
I'm also aware that it was probably already coded this way 🙂

Reply:
Please move the component in its own file. Blocking.